main.cpp

```diff
@@ -261,6 +259,23 @@ int Farm::payTaxes(int totalProfit)
     return totalProfit / 2 ;
 }
 
+int Farm::chickenMaximum(int chickens, float percentageIncreaseWeek )
+{ 
+    int weeks = 1;
+    int maximumChickens = acreage*50; // this is the maximum chickens we can fit
+    float x;
+    x = static_cast<float>(chickens);
+    while (chickens < maximumChickens)
+    { 
+    x = x + x*percentageIncreaseWeek;
+    chickens = static_cast<int>(x);    
+    ++ weeks; 
+    }
```

Comment:
Fix this indentation
Casting has not been covered yet
add space around `*` to make the code more readible
- [ ] revise

Reply:
removed the casting, but now we have compiler warnings (I actually only found out about casting after googling the float>int warnings  )